app/models/scc_account.rb

```diff
@@ -278,4 +285,18 @@ def invalidate_subscription_status(items_to_invalidate, save_record:)
     end
     items_to_invalidate
   end
+
+  def self.download_policy_selection_values
+    names = { 'on_demand' => _('On Demand'), 'immediate' => _('Immediate') }
```

Comment:
Looking at https://github.com/Katello/katello/blob/28a7171ed3be4fdb0ebd96bfad2304b05f0ea080/app/helpers/katello/concerns/smart_proxy_helper_extensions.rb#L28-L49 you should use constants such as `::Katello::RootRepository::DOWNLOAD_ON_DEMAND` and `::Katello::RootRepository::DOWNLOAD_IMMEDIATE`.